action.yml

```diff
@@ -47,6 +47,12 @@ inputs:
   cleanup:
     description: 'Cleanup temp files and remove builder at the end of a job'
     default: 'true'
+  keep-state:
+    description: 'Keep state on cleanup'
+    default: 'false'
+    required: false
+  custom-name:
```

Comment:
I'm not sure about naming this `custom-name`, this is confusing. Better to follow the buildx UX of the [create command](https://docs.docker.com/engine/reference/commandline/buildx_create/) and use `name`.